vpr/src/base/read_place.cpp

```diff
@@ -139,8 +139,8 @@ void read_place(const char* net_file,
     place_ctx.placement_id = vtr::secure_digest_file(place_file);
 }
 
-///@brief Reads in the locations of the IO pads from a file.
-void read_user_pad_loc(const char* pad_loc_file) {
+void read_user_block_loc(const char* constraints_file) {
```

Comment:
This takes a comment out of doxygen.  See https://www.doxygen.nl/manual/docblocks.html for how doxygen works.  Should put the comment before the function, using ///@brief so doxygen finds it.

Reply:
Should expand this comment.  Comment should include:
These locations can be for any subset of the cluster-level blocks, and they will be locked to the specified locations throughout placement. The format of the file is the same as the .place format.